src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
```

Comment:
alright, I've committed one with the .arg syntax, since that's what was suggested when I'd started. I can switch that over pretty easily though, if you'd like.